packages/authenticator/amplify_authenticator/lib/src/l10n/generated/button_localizations.dart

```diff
@@ -175,6 +183,12 @@ abstract class AuthenticatorButtonLocalizations {
   /// **'Skip'**
   String get skip;
 
+  /// Label of button to copy the TOTP key during TOTP setup.
+  ///
+  /// In en, this message translates to:
+  /// **'Copy Key'**
+  String get totpCopyKey;
```

Comment:
```suggestion
  String get copyKey;
```

Reply:
Basically, throughout, if the actual message has nothing to do with TOTP, then remove mention of it